application/Profile/LocalProfile.py

```
@@ -88,33 +94,41 @@
-            print('Not valid SSH key found for deletion')
+            logger.exception('No valid SSH key found for deletion')
 
         self._profile['sessions'].pop(session_id)
         self.save_profile()
 
+        logger.info("Successfully deleted session %s", session_id)
+
         return True, ''
 
-    def change_host(self, session_id, new_host):
+    def change_host(self, session_id, new_host) -> tuple[bool, str]:
```

Ruff (0.8.2)

Missing type annotation for function argument `session_id`

(ANN001)